### PR TITLE
fix: hidden state in redux

### PR DESCRIPTION
**Pull request by ShahAnuj2610**

### CODE REVIEW

File: src/utils/helper.js [hunk 1]
```diff
@@ -540,7 +540,7 @@ export function getResultStats(props) {
 		numberOfResults: total,
 		...(size > 0 ? { numberOfPages: Math.ceil(total / size) } : null),
 		time,
-		hidden,
+		hidden: hidden || 0,
```
Comment (jyash97) [inline src/utils/helper.js]: Can we put the fallback in `reducer` instead

Comment (ShahAnuj2610) [reply src/utils/helper.js]: done

File: src/actions/query.js [hunk 2]
```diff
@@ -253,6 +253,7 @@ function msearch(
 									component,
 									response.hits,
 									response.took,
+									response.hits.hidden,
```
Comment (bietkul) [inline src/actions/query.js]: @ShahAnuj2610 It'll break when `hits` is not defined, consider the error response.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: src/actions/query.js (hunk 2 above)
Comment (ShahAnuj2610) [reply src/actions/query.js]: @bietkul done

**Review by bietkul — APPROVED** [review]

**Pull request merged**